lms/static/sass/shared-v2/_variables.scss

```diff
@@ -0,0 +1,7 @@
+// LMS variables
+
+$lms-background-color: palette(grayscale, white);  // Why is the Pattern Library default background different?
```

Comment:
See the work that Brian did in the visual direction. https://drive.google.com/drive/folders/0B7jcFrBngjmkbG01ZVVzZjFVSkU